out/CI_Tester.js_sum_difference.js.html

```diff
@@ -45,8 +45,10 @@ <h1 class="page-title">Source: CI_Tester.js/sum_difference.js</h1>
  function mult(a, b) {
   return a * b;
 }
-
-module.exports = sum;</code></pre>
+//Want to test codacy
+alert("HELLO");
```

Comment:
Should this be removed

Reply:
I dont know what that is I didnt touch any of those files.

Reply:
I think it is an output from whatever they were doing with the ci